src/main/resources/db/migration/V20210413_6__CPO-6-UpdateCasePaymentOrders.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE public.case_payment_orders_audit
```

Comment:
Is this required ?

Reply:
I definitly wanted to get your eyes on this.  The `created_timestamp` value was not being populated as the `createdTimestamp` proeprrty is configured as `@NotAudited`.  Now that I have removed this anotation: it is requesting the mod field.

Reply:
Happy to get back to scenario where *everything* is audited